openhtf/io/user_input.py

```diff
@@ -84,7 +85,7 @@ def DisplayPrompt(self, message, text_input=False):
       console_prompt = ConsolePrompt(
           message, functools.partial(self.Respond, self.prompt.id))
       console_prompt.start()
-      self._cond.wait(int(FLAGS.prompt_timeout_s))
+      self._cond.wait(timeout_s or FLAGS.prompt_timeout_s)
```

Comment:
I think in order for the explicit ```None``` thing to work, we have to make the flag default ```None``` instead of 300 also. I think we might have talked about this earlier but I forget, so putting it here.